examples/php/oauthservice.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+function getClientCredentialsAccessToken()
+{
+    $token = $_SESSION["token"];
+    $expiry = $_SESSION["expiry"];
+    if(isset($token) && isset($expiry) && (time() < $expiry)) {
+        return $token;
+    }
+
+    /**
+     * Please change the following values to your uitgevers oauth credentials
+     * You could also use a oauth2 client library, but as our php.minversion = 5.3 we use curl.
+     * Preferably store your oauth-credentials in your default secure storage.
+     */
+    $clientid = '<client-id>';
+    $clientsecret = '<client-secret>';
+    $tokenhost = 'https://idp.toegang.org/token';
```

Comment:
idp-test dan hier ook
(maar liever nog definieren in een aparte config file...)